website/src/pages/HomePage/components/Hero.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export default function Hero() {
+  return (
+    <section className="flex items-center relative h-screen p-page">
+      <div className="absolute-cover overflow-hidden -z-1">
+        <div
+          className="absolute-cover bg-gradient-to-br from-primary to-accent scale-150"
+          style={{ clipPath: "polygon(0% 0%, 35% 0%, 62% 100%, 0% 100%)" }}
+        />
+      </div>
+      <div className="flex flex-1">
+        <div className="flex flex-col gap-y-8 flex-1 ml-8">
+          <div className="flex flex-col bg-background flex-1 p-10 rounded-3xl gap-y-2">
+            <h3 className="text-4xl font-semibold">Vitess Introduces</h3>
+            <h1 className="text-6xl font-bold text-primary">arewefastyet</h1>
+            <p className="text-lg font-normal mt-5 whitespace-nowrap">
+              A Cutting-Edge Benchmarking Approach for Unparalleled Database
+              Speed
+            </p>
```

Comment:
Let's remove the `Vitess Introduces` heading. And replace the slogan by: `A Benchmarking System for Vitess`